tests/foreman/ui/test_host.py

```diff
@@ -237,17 +262,25 @@ def module_libvirt_hostgroup(
 
 
 @pytest.fixture(scope='module')
-def module_activation_key(module_org_with_manifest, module_target_sat):
+def manifest_org(module_org):
+    """Upload manifest to organization."""
+    with manifests.clone() as manifest:
+        upload_manifest_locked(module_org.id, manifest.content)
+    return module_org
```

Comment:
This was moved a while back to taxonomies and renamed to `module_org_with_manifest`